src/core/util/util.js

```diff
@@ -83,12 +83,34 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
       return $options.getOption ? $options.getOption(optionName) : $options[optionName];
     },
 
+    /**
+     * Determines the current 'dir'ectional value based on the value of 'dir'
+     * attribute of the element. If that is not defined, it will try to use
+     * a 'dir' attribute of the body or html tag.
+     *
+     * @param $attrs
+     * @returns {boolean}
```

Comment:
```suggestion
     * @returns {boolean} true if the element's passed in attributes, the document, or the body indicates RTL mode, false otherwise.
```